api/v1/kogitoinfra_types.go

```diff
@@ -306,13 +364,17 @@ func (k KogitoInfraVolume) GetNamedVolume() api.ConfigVolumeInterface {
 // +kubebuilder:printcolumn:name="API Version",type="string",JSONPath=".spec.resource.apiVersion",description="Kubernetes CR API Version"
 // +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.condition.status",description="General Status of this resource bind"
 // +kubebuilder:printcolumn:name="Reason",type="string",JSONPath=".status.condition.reason",description="Status reason"
-// +operator-sdk:gen-csv:customresourcedefinitions.displayName="Kogito Infra"
-// +operator-sdk:gen-csv:customresourcedefinitions.resources="Kafka,kafka.strimzi.io/v1beta2,\"A Kafka instance\""
-// +operator-sdk:gen-csv:customresourcedefinitions.resources="Secret,v1,\"A Kubernetes Secret\""
+// +operator-sdk:csv:customresourcedefinitions:displayName="Kogito Infra"
+// +operator-sdk:csv:customresourcedefinitions:resources={{Kafka,kafka.strimzi.io/v1beta2,"A Kafka instance"}}
+// +operator-sdk:csv:customresourcedefinitions:resources={{Infinispan,infinispan.org/v1,"A Infinispan instance"}}
+// +operator-sdk:csv:customresourcedefinitions:resources={{Keycloak,keycloak.org/v1alpha1,"A Keycloak Instance"}}
```

Comment:
```suggestion
```

Reply:
with Kogito-infra integration in RHPAM kogito operator now we support them all